Source/WebCore/dom/FragmentDirectiveGenerator.cpp

```diff
@@ -141,6 +149,7 @@ void FragmentDirectiveGenerator::generateFragmentDirective(const SimpleRange& te
     if (suffix)
         components.append(makeString('-', *suffix));
 
+    const String textDirectivePrefix = ":~:text="_s;
```

Comment:
can this be a constexpr ASCIILIteral instead>

Reply:
again with the moving :) there's a lot in this code that could be much neater. but I'll see if I can make some of these changes

Reply:
`constexpr auto` if making the change.

Reply:
will `auto` know how to make an ASCIILiteral? guess we'll find out

Reply:
that is what `operator"" _s` returns so it sure should

Reply:
ah, I always mistakenly think it returns a String